File: 02. Programming fundamentals/14. Methods-Functions/04. orders.js
function orders(product, quantity) {
    let total = 0;

    switch (product) {
        case 'coffee':
            total = quantity * 1.50;
            break;
        case 'water':
            total = quantity * 1.00;
            break;
        case 'coke':
            total = quantity * 1.40;
            break;
        case 'snacks':
            total = quantity * 2.00;
            break;
    }

    console.log(total.toFixed(2));
}

orders("water", 5);

File: 06. Python Essentials/06. Functions/04. Orders.py
a = input()
b = int(input())


def calculate_price(product, qty):
    PRICE_LIST = {
        "coffee": 1.50,
        "water": 1.00,
        "coke": 1.40,
        "snacks": 2.00
    }

    return '{:.2f}'.format(PRICE_LIST[product] * qty)


print(calculate_price(a, b))
